source/agora/common/EnrollmentManager.d

```diff
@@ -74,7 +74,7 @@ public class EnrollmentManager
 
         // create the table for validator set if it doesn't exist yet
         this.db.execute("CREATE TABLE IF NOT EXISTS validator_set " ~
-            "(key BLOB PRIMARY KEY, val BLOB NOT NULL)");
+            "(key BLOB PRIMARY KEY, val BLOB NOT NULL, block_height INTEGER)");
```

Comment:
1. The name called `block_height` is kind of confusing. You could give clearer name to the column.
2. There is no description about 'block_height'. So we need it to understand what the column exactly means.

Reply:
Then what about the `enrolled_height`?

Reply:
It's good if the description is added. 'enrolled_block_height' is another option. :)